pkg/http/api_directmessaging.go

```diff
@@ -197,7 +197,7 @@ func (a *api) onDirectMessage(w http.ResponseWriter, r *http.Request) {
 // 1. HTTP header 'dapr-app-id' (path is method)
 // 2. Basic auth header: `http://dapr-app-id:<service-id>@localhost:3500/<method>`
 // 3. URL parameter: `http://localhost:3500/v1.0/invoke/<app-id>/method/<method>`
-func findTargetIDAndMethod(reqPath string, headers http.Header) (targetID string, method string) {
+func findTargetIDAndMethod(reqPath string, headers http.Header) (string, string) {
```

Comment:
Although we don't use named returns, it's useful to see in the signature what is being returned.

```suggestion
func findTargetIDAndMethod(reqPath string, headers http.Header) (targetID string, method string) {
```